### PR TITLE
[[ HTML5Callbacks ]] Allow specified LiveCode handlers to be called from JavaScript

### PR DESCRIPTION
**Pull request by livecodeian**

### CODE REVIEW

File: docs/notes/feature-html5-callbacks.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+**Note** This is still an experimental feature - details may change as development continues.
+ 
+## Stack setup
+ 
+Each stack can be configured to expose its handlers & functions to JavaScript. This is done through a custom property of the stack - `cJavascriptHandlers`. (This will be replaced in the future by a `javascriptHandlers` property which will then be a reserved keyword). The `cJavascriptHandlers` property is a return-delimited list of handler names. The named handlers do not have to be defined at the time the stack is loaded, however calling an undefined handler from JavaScript will result in an error.
+ 
+ 
+## Calling from JavaScript
+ 
+The standalone engine will create a `liveCode` object on the DOM `document` object. To this object will be attached the `findStackWithName` method that can be called to return a JavaScript stack object. Each stack object will have methods corresponding to the exposed handlers of that stack. For instance, a stack with the `cJavascriptHandlers` property set to :
```
Comment (runrevmark) [inline docs/notes/feature-html5-callbacks.md:12]: I suggest we make the name 'livecode' - I think liveCode will be prone to errors.

Comment (livecodeian) [reply docs/notes/feature-html5-callbacks.md:12]: liveCode is consistent with the naming in the browser widget, although the usage there is slightly different

Comment (runrevmark) [reply docs/notes/feature-html5-callbacks.md:12]: that's fine - we'll keep it consistent for now - we can add an alias at a later date.

**Comment by runrevmark** [comment]
@livecode-vulcan review ok 93f1d75

**Comment by livecode-vulcan** [comment]
💙 review by @runrevmark ok 93f1d7587cd1ddb7299c933dd9568668c8ff3344

**Comment by livecode-vulcan** [comment]
😎 test success 93f1d7587cd1ddb7299c933dd9568668c8ff3344

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/380)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/386)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/366)
* try-community-universal-ios-iphoneos10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos10.3/builds/309)
* try-community-universal-ios-iphonesimulator10.3: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator10.3/builds/309)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/375)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/380)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/378)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/384)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/378)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/384)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/403)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/374)

**Comment by mwieder** [comment]
This is an awesome addition. I am so looking forward to this feature. Thanks.

**Pull request merged**